test/logger-agent-log.js

```diff
@@ -368,3 +368,76 @@ test('.agentLog() warns if line is blank', async (t) => {
     })
   })
 })
+
+test('.agentLog() payload format handles 207 partial-success responses and emits errors', (t) => {
```

Comment:
Correct, but since the payload structures are somewhat different, I wanted to test in both places.  If `line` was somehow not a part of the agent payload, it would throw.  Seemed thorough to just copy the test and use it for `agentLog`

Reply:
ah I see now, makes sense 👍